concordium-contracts-common/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased changes
 
+## concordium-contracts-common 4.1.0 (Unreleased)
```

Comment:
This should still be under "Unreleased changes". The process is that I will tag this with the version when I do a release on crates.io.